src/memory_pool.c

```diff
@@ -278,6 +285,10 @@ static umf_result_t umfPoolCreateInternal(const umf_memory_pool_ops_t *ops,
         pool->ops.ext_ctl = umfDefaultCtlPoolHandle;
     }
 
+    if (NULL == pool->ops.ext_trim_memory) {
+        pool->ops.ext_trim_memory = umfDefaultTrimMemory;
+    }
+
     if (NULL == utils_mutex_init(&pool->lock)) {
```

Comment:
Read/write to undefined memory if user gives us old ops version

Reply:
done